packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md

```diff
@@ -0,0 +1,37 @@
+
+| Type      | Purpose                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Primary   | Primary Buttons are used to enable the most important actions, that is those that lead to the completion of the main task (e.g., Save, Continue, Submit), or strong call to actions. Ideally, there should be only one Primary Button per screen to avoid dilluting the user's attention.                                                                                                                                                                                                  |
+| Secondary | Secondary Buttons are used for main actions that users might want to execute but that are not in direct relation to the main task that is to be done in the screen or interface.                                                                                                                                                                                                                                                                                                           |
+| Combo     | The Combo Button got an attached drop menu because a choice of action has to be made (2 or more). They are usually used when one single action allows to create, delete or modify many different elements, which can be chosen in the dropdown list that appears on click. While using a combo button allows to avoid adding an extra step in the process, it can add friction to the user's decision making process if the options in the dropdown list are not intuitive enough to them. |
+| Isolated  | Isolated Action Buttons are used for extra actions located on their own within an interface. Button borders and background color are removed to reduce the noise generated by the presence of many consecutive borders and surfaces, or to make it more discrete in the screen. The icon is optional and aims to help users identify the action faster. However, keep in mind that too many icons in a screen can be as distracting and confusing as too less.                             |
```

Comment:
Suggestion:
**Isolated Action Buttons** enable extra actions which are, as the name suggests, isolated in an interface. Those button do not have borders and background colors, in order to reduce noise and to make them less conspicuous. Icons are optional and may help users identify actions faster. However, keep in mind that too many icons can be as distracting and confusing as too few.